examples/practice-typescript/src/styles/component.scss

```diff
@@ -2,6 +2,7 @@
 @import '../components/Input/index.scss';
 @import '../components/Button/index.scss';
 @import '../components/SideBar/index.scss';
+@import '../components/TableHeader\ /';
```

Comment:
Recheck the path

Reply:
Fixed

Reply:
The path still not resolved. Please check